### PR TITLE
[bugfix/deps]: Unrelated Energy Error on Tron Bandwidth Unfreeze + Update Wallet API packages

### PR DESCRIPTION
**Pull request by Philippoes**
### 📝 Description

This PR resolves an issue when unfreezing bandwidth: a TronNoFrozenForEnergy() error was thrown, even though energy amounts should not be a factor in this context. We've fixed this by implementing an additional check to confirm the transaction's resource property is marked as "ENERGY" before proceeding with any energy balance verification.

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Comment by Philippoes** [comment]
Hey @haammar-ledger, any chance you can have a look at this PR? This is the final piece for us to go live with our Tron integration on our Stakekit Live App 🙂

File: libs/ledger-live-common/src/families/tron/bridge/js.ts [hunk 1]
```diff
@@ -649,7 +649,7 @@ const getTransactionStatus = async (a: TronAccount, t: Transaction): Promise<Tra
     const { bandwidth, energy } = a.tronResources.frozen;
     if (resource === "BANDWIDTH" && t.amount.gt(bandwidth?.amount || new BigNumber(0))) {
       errors.resource = new TronNoFrozenForBandwidth();
-    } else if (t.amount.gt(energy?.amount || new BigNumber(0))) {
+    } else if (resource === "ENERGY" && t.amount.gt(energy?.amount || new BigNumber(0))) {
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/tron/bridge/js.ts:652]: Do you think you could cover this with a case in https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/families/tron/bridge.integration.test.ts ?

Comment (Philippoes) [reply libs/ledger-live-common/src/families/tron/bridge/js.ts:652]: @haammar-ledger just added the test. Let me know if it's sufficient. 🙂

**Comment by Philippoes** [comment]
@haammar-ledger I added the test and have rebased the branch (I saw the Wallet API packages already got bumped in develop). Can you check this out and let me know if it looks good to you? 🙂

**Review by haammar-ledger — APPROVED** [review]
Just pending CI & QA approval

**Pull request merged**